docs/docs/tutorials/orca-yarn-tutorial.md

```diff
@@ -0,0 +1,532 @@
+```python
+from bigdl.orca import init_orca_context
+
+init_orca_context(cluster_mode="yarn-client", cores=4, memory="10g", num_nodes=2, 
+                  driver_cores=2, driver_memory="4g",
+                  extra_python_lib="model.py")
+```
+__Note:__
+* Jupyter Notebook cannot run on `yarn-cluster`, as the driver is not running on the __Client Node__(the notebook page).
+
+## 5.2 Use `bigdl-submit`
+For users who want to use a script instead of Python command, BigDL provides an easy-to-use `bigdl-submit` script (under `${BIGDL_HOME}/scripts`), which could automatically setup configuration and jars files from the current activate Conda environment.
+
+Please call `init_orca_context` at the very beginning of the program.
+```python
+from bigdl.orca import init_orca_context
+
+if args.cluster_mode == "bigdl-submit":
+    init_orca_context(cluster_mode="bigdl-submit")
+```
+
+On the __Client Node__ (where you submit applications), before submitting the example:
+1. Install and activate Conda environment (see __Section 2.1__).
+2. Use Conda to install BigDL and other Python libraries (see __Section 2.2__).
+3. Pack the current activate Conda environment to an archive.
+    ```bash
+    conda pack -o environment.tar.gz
+    ```
+
+### 5.2.1 Yarn Client
+Submit and run the example on `yarn-client` mode following `bigdl-submit` script below:
+```bash
+bigdl-submit \
+    --master yarn \
+    --deploy-mode client \
+    --executor-memory 10g \
+    --driver-memory 10g \
+    --executor-cores 8 \
+    --num-executors 2 \
+    --py-files model.py \
+    --archives /path/to/environment.tar.gz#environment \
+    --conf spark.pyspark.driver.python=/path/to/python \
+    --conf spark.pyspark.python=environment/bin/python \
+    train.py --cluster_mode bigdl-submit --remote_dir hdfs://url:port/path/to/remote/data
+```
+In the `bigdl-submit` script:
+* `--master`: the spark master, set it to yarn;
+* `--deploy-mode`: set it to client when running programs on yarn-client mode;
+* `--executor-memory`: set the memory for each executor;
+* `--driver-memory`: set the memory for the driver node;
+* `--executor-cores`: set the cores number for each executor;
+* `--num_executors`: set the number of executors;
+* `--py-files`: upload extra Python dependency files to YARN;
+* `--archives`: upload the Conda archive to YARN;
+* `--conf spark.pyspark.driver.python`: set the activate Python location on __Client Node__ as driver's Python environment;
+* `--conf spark.pyspark.python`: set the Python location in Conda archive as executors' Python environment;
+
+__Note:__
+* `--cluster_mode`: set the cluster_mode in `init_orca_context`.
+* `--remote_dir`: directory on a distributed storage for the dataset (see __Section 3__) and saving the model.
+* Please refer to __Section 4__ for the description of extra Python dependencies.
+* For CDH Users, please speficy `ARROW_LIBHDFS_DIR` (see __Section 2.3__) as a Spark conf in the script.
```

Comment:
If we can remove pyarrow dependency, can we remove such related stuff in this tutorial?